scripts/clan_package/settings/clan_settings.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import ujson
+
+from scripts.game_structure.game.save_load import safe_save
+from scripts.game_structure.game.switches import Switches, get_switch
+from scripts.housekeeping.datadir import get_save_dir
+
+
+def load_clan_settings():
+    reset_loaded_clan_settings()
+    if os.path.exists(
+        get_save_dir() + f"/{get_switch(Switches.clan_list)[0]}/clan_settings.json"
```

Comment:
Moving forward, we should try to use Pathlib instead of constructing path literals as strings.

Reply:
I was aiming to change as little as possible in this to make it easier to review but I see your point 😂 these come direct from the original code. Want me to change all those instances to pathlib?